### ISSUE TITLE
Load Entrypoint Plugins Before Directory Plugins

### ISSUE THREAD

**Issue by aplett98** [apache/airflow#16640]
https://github.com/apache/airflow/blob/ffb1fcacff21c31d7cacfbd843a84208fca38d1e/airflow/plugins_manager.py#L291-L292
Currently, entrypoint plugins are loaded after plugins directory plugins. If there's not a specific reason for this ordering, I think these should be reversed.

For example, if I have a custom operator `MyOperator` that I want to derive from another developer's custom operator `TheirOperator`, and that operator is part of a package that uses the Airflow plugins entrypoint, I would probably have a `my_operator.py` file that looks like this:
```python3
from airflow.operators.their_plugin import TheirOperator

class MyOperator(TheirOperator):
    ...
    ...
```
As it stands, since `MyOperator` is loaded first, this will fail with `ModuleNotFoundError` on `their_package.plugins`, making it difficult to derive operators from other packages.

**Comment by uranusjr** [comment]
You can close this issue if you feel #16641 covers the topic better—or close #16641 otherwise, just keep one open, either way.

**Comment by gunplun** [comment]
Closed because duplicated by #16641
